src/Sonarr.Api.V3/Update/UpdateController.cs

```diff
@@ -45,7 +48,11 @@ public List<UpdateResource> GetRecentUpdates()
                     installed.Installed = true;
                 }
 
-                var installDates = _updateHistoryService.InstalledSince(resources.Last().ReleaseDate)
+                var updateHistory = _configFileProvider.LogDbEnabled
```

Comment:
This can avoid looping through all the updates by wrapping the existing logic in `if (_configFileProvider.LogDbEnabled)`

Reply:
Good spot